src/web/modules/payment_links/detail.njk

```diff
@@ -0,0 +1,122 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "common/json.njk" import json %}
+{% extends "layout/layout.njk" %}
+
+{% block main %}
+  <span class="govuk-caption-m">{{ account.description or account.gateway_account_id }}</span>
```

Comment:
Looks like nothing called account is ever passed in to this so probably okay to remove it or if we wanted to add the context for the payment link, quickly fetching the service and using service name would probably be the best title